quilt_lite/quilt_lite/s3proxy.py

```diff
@@ -0,0 +1,74 @@
+"""
+Python implementation of the s3 proxy from quilt/s3-proxy/nginx.conf
+"""
+import urllib.parse
+
+import aiohttp
+from fastapi import FastAPI, Request, Response
+
+
+s3proxy = FastAPI()
+
+session = aiohttp.ClientSession()
```

Comment:
don't we want a new session for every request?

Reply:
Possibly... Not sure. In general, re-using a session is good for performance - but it might not be correct in this case. I guess I'll change it for now, until we know what's safe.